librz/il/definitions/mem.c

```diff
@@ -0,0 +1,79 @@
+// SPDX-FileCopyrightText: 2021 heersin <teablearcher@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_il/definitions/mem.h>
+
+static void free_bv_key_value(HtPPKv *kv) {
+	rz_il_bv_free(kv->value);
+	rz_il_bv_free(kv->key);
+}
+
+/**
+ * Create a Mem (Array)
+ * \param min_unit_size, minimal size of a data unit of current arch
+ * \return RzILMem
+ */
+RZ_API RzILMem rz_il_new_mem(int min_unit_size) {
```

Comment:
Can be negative? isn't better to use unsigned ? like ut32 ?

Reply:
better to be unsigned number